python/ray/_raylet.pyx

```diff
@@ -1854,7 +1854,8 @@ cdef execute_task_with_cancellation_handler(
 
     # Automatically restrict the GPUs (CUDA), neuron_core, TPU accelerator
     # runtime_ids to restrict availability to this task.
-    ray._private.utils.set_visible_accelerator_ids()
+    if (<int>task_type != <int>TASK_TYPE_ACTOR_TASK):
```

Comment:
Can you add a comment for this if statemenet (once actor is created, users can change the cuda visible devices within an actor task, and we don't want to reset it in system layer)